cinder/README.rst

```diff
@@ -17,72 +17,40 @@ The VNX container image contains following RPM packages:
 * python-retryz
 
 
-.. attention::
-
-  Cinder containerization is still in `technical preview only <https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/12/html/release_notes/containers>`_ for OSP12.
-
-
 Deployment prerequisites
 ------------------------
 
-* RedHat OpenStack Platform 12 with Cinder containerization (it is disabled by default).
+* RedHat OpenStack Platform 13.
 * VNX with Block version 5.32 or above.
 
 Deployment steps
 ----------------
 
 Prepare Dell EMC container
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
-Before starting the deployment with VNX driver, User needs to build and push the Dell EMC container image into the local docker registry.
-There are 2 options to do it: 1) `From the Dockerfile`_, 2) `From the binary container image`_.
-
-.. attention::
+The formal Dell EMC container image is published to `Red Hat Container Catalog <https://access.redhat.com/containers/>`_.
 
-    in below examples, the 192.168.139.1:8787 acts as the local registry.
-
-From the Dockerfile
-^^^^^^^^^^^^^^^^^^^
-The recommended way is to build via a standalone `Dockerfile` if the overcloud is able to access the Internet.
-
-- First, download the `Dockerfile <./Dockerfile>`_ to the director, and build a local image:
+Frist, pull the container image from Red Hat Container Catalog.
 
 .. code-block:: bash
 
-  $ docker build -t 192.168.139.1:8787/rhosp12/openstack-cinder-volume-dellemc .
-
-Above command will fetch and install all the VNX dependencies from Dell EMC repos.
+  $ docker pull registry.connect.redhat.com/dellemc/openstack-cinder-volume-dellemc
 
-- Then, push the image to the local docker registry.
+Then, tag and push it to the local docker registry if needed.
 
 .. code-block:: bash
 
-  $ docker push 192.168.139.1:8787/rhosp12/openstack-cinder-volume-dellemc
-
-
-From the binary container image
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  $ docker tag registry.connect.redhat.com/dellemc/openstack-cinder-volume-dellemc 192.168.139.1:8787/dellemc/openstack-cinder-volume-dellemc
+  $ docker push 192.168.139.1:8787/dellemc/openstack-cinder-volume-dellemc
 
-- First, download the binary `dellemc-container.tar <https://github.com/emc-openstack/osp-deploy/releases/download/0.0.1/dellemc-container.tar>`_ to the director,
-
-.. code-block:: bash
-
-  $ docker load -i ./dellemc-container.tar
-  Loaded image ID: sha256:<IMAGE ID>
-
-This returns an **<IMAGE ID>** for later use.
-
-- Then, tag and push it to the local docker registry.
-
-.. code-block:: bash
+.. attention::
 
-  $ docker tag <IMAGE ID> 192.168.139.1:8787/rhosp12/openstack-cinder-volume-dellemc
-  $ docker push 192.168.139.1:8787/rhosp12/openstack-cinder-volume-dellemc
+  in below examples, the 192.168.139.1:8787 acts as the local registry.
```

Comment:
remove "the"
192.168.139.1:8787 is shown above, I think the description here should be moved before it.

Reply:
Done